utils/generate/schema.json

```diff
@@ -371,6 +371,22 @@
       "ct-validate": "",
       "ct-fieldgroups": ["SecurityScanner"]
     },
+    "SECURITY_SCANNER_V4_SIGN_JWT": {
+      "type": "boolean",
+      "description": "Whether ClairV4 client will sign jwts.",
+      "x-example": false,
+      "ct-default": false,
+      "ct-validate": "",
+      "ct-fieldgroups": ["SecurityScanner"]
+    },
+    "SECURITY_SCANNER_V4_PSK": {
+      "type": "string",
+      "description": "If 'SECURITY_SCANNER_V4_SIGN_JWT' true, whether to use psk or Quay key server. If PSK is desired define the key in this config variable.",
```

Comment:
this reads a little funny, and I'm sure you got it from my doc string originally because i don't English well :-p.

description: "If 'SECURITY_SCANNER_V4_SIGN_JWT' Quay will sign JWTs with either the key provided by `SECURITY_SCANNER_V4_PSK' or the Quay instance's private key otherwise."

something like this.

Reply:
changed to "If 'SECURITY_SCANNER_V4_SIGN_JWT', Quay will sign JWTs with either the key provided by `SECURITY_SCANNER_V4_PSK' (if specified here) or the Quay instance's private key otherwise."